packages/sdk/src/accountTransactions.ts

```diff
@@ -51,6 +52,8 @@ export interface AccountTransactionHandler<
     PayloadType extends AccountTransactionPayload = AccountTransactionPayload,
     JSONType = PayloadType,
 > {
+    create(metadata: TransactionMetadata, payload: PayloadType, givenEnergy?: Energy.Type): AccountTransaction;
```

Comment:
Add docs. Also, maybe this should be formatted more like the other operations:

```suggestion
    create: (metadata: TransactionMetadata, payload: PayloadType, givenEnergy?: Energy.Type) => AccountTransaction;
```